src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/UserActivityOverview.html

```diff
@@ -0,0 +1,7 @@
+<div class="user-activity-overview">
+  <div class="wrapper">
+    <div class="comments" data-ng-if="commentCount">{{ commentCount }} {{ "TR__COMMENTS" | translate}}</div>
+    <div class="proposals" data-ng-if="proposalCount">{{ proposalCount }} {{ "TR__PROPOSALS" | translate }}</div>
+    <div class="ratings" data-ng-if="rateCount">{{ rateCount }} {{ "TR__RATES" | translate}}</div>
```

Comment:
These classes should be prefixed with widget name (see [CSS guidelines](https://adhocracy3mercator.readthedocs.org/en/latest/coding_guides/css_guidelines.html#selectors))